tools/sklearn/model_validation.xml

```diff
@@ -51,63 +55,119 @@ y = read_columns(
 )
 y=y.ravel()
 
-validator = params["model_validation_functions"]["selected_function"]
-validator = getattr(sklearn.model_selection, validator)
 options = params["model_validation_functions"]["options"]
 if 'scoring' in options and options['scoring'] == '':
     options['scoring'] = None
+if 'pre_dispatch' in options and options['pre_dispatch'] == '':
```

Comment:
`if options.get('pre_dispatch', False):`

Reply:
A little tricky here. I'll be happy if you can make it better.